sw/airborne/modules/nav/nav_bungee_takeoff.c

```diff
@@ -22,198 +22,183 @@
 /**
  * @file modules/nav/nav_bungee_takeoff.c
  *
+ * Takeoff functions for bungee takeoff.
+ *
  * from OSAM advanced navigation routines
  *
  */
 
 #include "modules/nav/nav_bungee_takeoff.h"
 
-#include "firmwares/fixedwing/nav.h"
 #include "state.h"
-#include "autopilot.h"
+#include "paparazzi.h"
+#include "firmwares/fixedwing/nav.h"
+#include "firmwares/fixedwing/autopilot.h"
 #include "generated/flight_plan.h"
+#include "generated/airframe.h"
+#include "math/pprz_algebra_float.h"
 
 /************** Bungee Takeoff **********************************************/
 
-/** Takeoff functions for bungee takeoff.
-    Run initialize function when the plane is on the bungee, the bungee is fully extended and you are ready to
-    launch the plane. After initialized, the plane will follow a line drawn by the position of the plane on initialization and the
-    position of the bungee (given in the arguments). Once the plane crosses the throttle line, which is perpendicular to the line the plane is following,
-    and intersects the position of the bungee (plus or minus a fixed distance (TakeOff_Distance in airframe file) from the bungee just in case the bungee doesn't release directly above the bungee) the prop will come on. The plane will then continue to follow the line until it has reached a specific
-    height (defined in as Takeoff_Height in airframe file) above the bungee waypoint and speed (defined as Takeoff_Speed in the airframe file).
-    @verbatim
-    <section name="Takeoff" prefix="Takeoff_">
-    <define name="Height" value="30" unit="m"/>
-    <define name="Speed" value="15" unit="m/s"/>
-    <define name="Distance" value="10" unit="m"/>
-    <define name="MinSpeed" value="5" unit="m/s"/>
-    </section>
-    @endverbatim
-*/
-
-#ifndef Takeoff_Distance
-#define Takeoff_Distance 10
+/**
```

Comment:
Would make sense to put this description in the doxygen file header so it is put in the detailed description in the generated docs....